public/app/plugins/panel/alertlist/AlertInstances.tsx

```diff
@@ -48,6 +64,47 @@ export const AlertInstances = ({ alerts, options }: Props) => {
     }
   }, [filteredAlerts]);
 
+  const onShowAllClick = async () => {
+    if (!handleInstancesLimit) {
+      return;
+    }
+    await handleInstancesLimit(false);
```

Comment:
I don't think the prop is specified as a `(): Promise<T>` so why do we need to `await` here?

Reply:
💯